helm/charts/determined/templates/change-password.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: determined-master-pw-change-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: determined-master-pw-change
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
```

Comment:
I would probably get rid of these labels, they are good practice but we don't follow this pattern in the other templates right now and instead add:
```
app: determined-pw-change-{{ .Release.Name }}
release: {{ .Release.Name }}
```